lib/deploy.js

```diff
@@ -33,6 +34,20 @@ module.exports = promisify(function(config, callback) {
   var rootDir = config.rootDir ? config.rootDir : '.';
   var cloneDir = config.cloneDir ? config.cloneDir : null;
 
+  if (config.fileGlobs) {
+    fs.mkdirSync('tmp');
```

Comment:
It wasn't possible in the first implementation of my patch (because I was using the *src* option, which accepts a relative directory), but it is possible now.
Done in 0f5c102.